server/test/helper.js

```diff
@@ -3,12 +3,23 @@
 
 import helper from 'fastify-cli/helper.js';
 import path from 'path';
+import * as nodemailerMock from 'nodemailer-mock';
+import quibble from 'quibble';
 import { fileURLToPath } from 'url';
 
+import { PrismaClient } from '@prisma/client';
+
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = path.dirname(__filename);
 const AppPath = path.join(__dirname, '..', 'app.js');
 
+// Environment variable overrides/additions for testing
+process.env.NODE_ENV = 'test';
+process.env.DATABASE_URL = `${process.env.DATABASE_URL}_test`;
+
+// Dependency mocks for testing
+await quibble.esm('nodemailer', { default: nodemailerMock });
```

Comment:
I've added the quibble library which lets us override dependencies with mock replacements. Here, I replace nodemailer with nodemailer-mock so that emails are not actually sent but captured for inspection by the mock.